app/src/main/java/com/github/polypoly/app/ui/menu/lobby/JoinGameLobbyActivity.kt

```diff
@@ -348,7 +342,8 @@ class JoinGameLobbyActivity : MenuActivity("Join a game") {
             Text(
                 text = "Players:",
                 style = MaterialTheme.typography.h6,
-                fontSize = 16.sp
+                fontSize = 16.sp,
```

Comment:
It could be nice to have constants in the `utils` package for the fontSizes and use only those, so if we want to increase the text size of the whole application (for example to make a tablet version) we can.

Reply:
agreed. I will put it as an issue for someone to carefully gather and merge constants throughout the code :)

Reply:
actually there is [this issue](https://github.com/polypoly-team/android-app/issues/40) that should be adressed, and should solve this case